sdk/kusto/arm-kusto/test/kusto-client-test.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import {
+  env,
+  record,
+  RecorderEnvironmentSetup,
+  Recorder,
+  isPlaybackMode
+} from "@azure-tools/test-recorder";
+import { Cluster, ClustersGetOptionalParams, ClustersListOptionalParams, ClusterUpdate, KustoManagementClient } from "../src";
+import { ClientSecretCredential } from "@azure/identity";
+import * as assert from "assert";
+
+const recorderEnvSetup: RecorderEnvironmentSetup = {
+  replaceableVariables: {
+    AZURE_CLIENT_ID: "azure_client_id",
+    AZURE_CLIENT_SECRET: "azure_client_secret",
+    AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
+    SUBSCRIPTION_ID: "azure_subscription_id"
+  },
+  customizationsOnRecordings: [
+    (recording: any): any =>
+      recording.replace(
+        /"access_token":"[^"]*"/g,
+        `"access_token":"access_token"`
+      )
+  ],
+  queryParametersToSkip: []
+};
+
+export const testPollingOptions = {
+  updateIntervalInMs: isPlaybackMode() ? 0 : undefined,
+};
+
+describe("KustoManagementClient", () => {
+  let recorder: Recorder;
+  let subscriptionId: string;
+  let client: KustoManagementClient;
+  let resourceGroup: string;
+  let clusterName_1: string;
+  let clusterName_2: string;
+  let clusterParameters: Cluster;
+
+  beforeEach(async function () {
+    recorder = record(this, recorderEnvSetup);
+    subscriptionId = env.SUBSCRIPTION_ID;
+    // This is an example of how the environment variables are used
+    const credential = new ClientSecretCredential(
+      env.AZURE_TENANT_ID,
+      env.AZURE_CLIENT_ID,
+      env.AZURE_CLIENT_SECRET
+    );
+    client = new KustoManagementClient(credential, subscriptionId);
+    resourceGroup = "marytest";
+    clusterName_1 = "mytestclustername5";
+    clusterName_2 = "mytestclustername6";
+    clusterParameters = {
+      "location": "westeurope",
+      "sku": {
+        "name": "Standard_L8s_v2",
+        "tier": "Standard"
+      },
+      "identity": {
+        "type": "SystemAssigned"
+      },
+    };
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  //kusto_client.clusters.beginCreateOrUpdateAndWait
+  it("could create clusters", async function () {
+    let res = await client.clusters.beginCreateOrUpdateAndWait(resourceGroup, clusterName_1, clusterParameters, testPollingOptions);
+    assert.strictEqual(res.name, clusterName_1);
+    res = await client.clusters.beginCreateOrUpdateAndWait(resourceGroup, clusterName_2, clusterParameters, testPollingOptions);
+    assert.strictEqual(res.name, clusterName_2);
```

Comment:
why do you need to call the create twice ?

Reply:
to create two clusters so that easy to get by cluster name